### PR TITLE
Merge Features to Dev Branch

### PR DESCRIPTION
**Pull request by MephistoDevelop**
Features Created:
- Add Webpack
- Add Module pathern
- Add Inmediate invoked functions
- Dynamical UI
- Localstorage
- Todo Task Logic

### CODE REVIEW

**Comment by MephistoDevelop** [comment]
Sure, i will refactor the code and send a new review with the changes , thanks !!.

**Comment by MephistoDevelop** [comment]
Fixed  changes from Code review:
- Add Factory funcions
- Separate Trasks

File: src/DOMmodule.js [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+    textAdd.innerText = '+ Add List';
+    textAdd.id = 'add-text';
+
+    mainContainer.appendChild(navContainer);
+    buttonAddContainer.appendChild(textAdd);
+    navContainer.appendChild(titleNav);
+    navContainer.appendChild(buttonAddContainer);
+    navContainer.appendChild(listContainer);
+    rowContainer.appendChild(navContainer);
+    // textAdd.onclick = showMenuAdd;
```
Comment (bellom) [inline src/DOMmodule.js]: kindly remove this comment.

Comment (MephistoDevelop) [reply src/DOMmodule.js]: Sure i will do that before merge !!, thanks!

**Pull request merged**